pages/deploy/deploy_aws.md

```diff
@@ -79,6 +80,27 @@ git add .
 git commit -m "First commit"
 git push --set-upstream origin main
 ```
+### Using GitHub:
+If you choose to use a GitHub repository, it's important to note that you need to set up an AWS CodeStar connection to your GitHub repository for seamless integration. 
```

Comment:
Can we add a link to the documentation? I think it will help because CodeStar console is not where the users create the connection, it is directly in the CICD-suite console under connections

Reply:
Fixed